Description: Sets the default number of channels for module-jackdbus-detect to two
Author: Kaj Ailomaa <zequence@mousike.me>

--- pulseaudio-3.0.orig/src/daemon/default.pa.in
+++ pulseaudio-3.0/src/daemon/default.pa.in
@@ -78,7 +78,7 @@ load-module module-detect
 ### Automatically connect sink and source if JACK server is present
 .ifexists module-jackdbus-detect@PA_SOEXT@
 .nofail
-load-module module-jackdbus-detect
+load-module module-jackdbus-detect channels=2
 .fail
 .endif
 
